#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_arch.man.dpatch by Michael Stone <mstone@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: man page isn't built in upstream tar, and prevents clean build

@DPATCH@
--- coreutils-8.13.orig/man/arch.1
+++ coreutils-8.13/man/arch.1
@@ -0,0 +1,47 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.35.
+.TH ARCH "1" "September 2011" "GNU coreutils 8.13" "User Commands"
+.SH NAME
+arch \- print machine hardware name (same as uname -m)
+.SH SYNOPSIS
+.B arch
+[\fIOPTION\fR]...
+.SH DESCRIPTION
+.\" Add any additional description here
+.PP
+Print machine architecture.
+.TP
+\fB\-\-help\fR
+display this help and exit
+.TP
+\fB\-\-version\fR
+output version information and exit
+.SH AUTHOR
+Written by David MacKenzie and Karel Zak.
+.SH "REPORTING BUGS"
+Report arch bugs to bug\-coreutils@gnu.org
+.br
+GNU coreutils home page: <http://www.gnu.org/software/coreutils/>
+.br
+General help using GNU software: <http://www.gnu.org/gethelp/>
+.br
+Report arch translation bugs to <http://translationproject.org/team/>
+.SH COPYRIGHT
+Copyright \(co 2011 Free Software Foundation, Inc.
+License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
+.br
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+.SH "SEE ALSO"
+uname(1), uname(2)
+.PP
+The full documentation for
+.B arch
+is maintained as a Texinfo manual.  If the
+.B info
+and
+.B arch
+programs are properly installed at your site, the command
+.IP
+.B info coreutils \(aqarch invocation\(aq
+.PP
+should give you access to the complete manual.
